pipelines/pipelines/githubapp-copyright/build-branch.yaml

```diff
@@ -72,4 +75,46 @@ spec:
       value: ""
     workspaces:
      - name: git-workspace
-       workspace: git-workspace
\ No newline at end of file
+       workspace: git-workspace
+
+  # 
+  # Assemble into a container image.
+  #
+  - name: recycle-deployment
+    taskRef:
+      name: argocd-cli
+    runAfter:
+    - docker-build-githubapp-copyright
+    params:
+    - name: server
+      value: argocd.galasa.dev
+    - name: command
+      value: 
+      - app 
+      - actions 
+      - run 
+      - $(params.appnameCli)
+      - restart 
+      - --kind 
+      - Deployment
+      - --resource-name
+      - cli-$(params.imageTag)
```

Comment:
This would be the resource name, so the name of the Deployment - 'githubappcopyright'.